sigridci/sigridci-feedback-template.html

```diff
@@ -254,7 +254,7 @@ <h3>System property</h3>
                             <span class="separator">Maintainability:</span>
                         </div>
                         <div>
-                            <h3>Overall quality</h3>
+                            <h3>Baseline (@@@BASELINE)</h3>
```

Comment:
I'd call this @@@BASELINE_DATE as initially I thought this to be a star rating given the next few lines